static/js/navbar_alerts.js

```diff
@@ -196,8 +196,7 @@ export function initialize() {
 export function open(args, rendered_alert_content_html) {
     const rendered_alert_wrapper_html = $(render_navbar_alert_wrapper(args));
 
-    // TODO: Use a proper CSS class to access correct outer widget.
-    rendered_alert_wrapper_html.first().prepend(rendered_alert_content_html);
+    rendered_alert_wrapper_html.prepend(rendered_alert_content_html);
```

Comment:
I'm not sure this does what you want -- I think this will put the content outside what is meant to be the other div.

I think what you should actually be doing is passing in the rendered_alert_content_html as an HTML parameter into the wrapper template.